### PR TITLE
Fix list format of traffic-management/request-timeouts/index.md

### PR DESCRIPTION
**Pull request by wilsonwu**
Fix list format of traffic-management/request-timeouts/index.md:
```
content/en/docs/tasks/traffic-management/request-timeouts/index.md
```

- [ ] Ambient
- [x] Docs
- [ ] Installation
- [ ] Networking
- [ ] Performance and Scalability
- [ ] Extensions and Telemetry
- [ ] Security
- [ ] Test and Release
- [ ] User Experience
- [ ] Developer Infrastructure

### CODE REVIEW

File: content/en/docs/tasks/traffic-management/request-timeouts/index.md [hunk 1]
```diff
@@ -29,7 +29,7 @@ timeout to half a second.
 To see its effect, however, you also introduce an artificial 2 second delay in calls
 to the `ratings` service.
 
-1.  Route requests to v2 of the `reviews` service, i.e., a version that calls the `ratings` service:
+1)  Route requests to v2 of the `reviews` service, i.e., a version that calls the `ratings` service:
```
Comment (windsonsea) [inline content/en/docs/tasks/traffic-management/request-timeouts/index.md:32]: In my understanding, `1.` is correct before `{{< tabset>}}`, and 2), 3)... will be used after it in sequence. Need a confirmation :)

Comment (wilsonwu) [reply content/en/docs/tasks/traffic-management/request-timeouts/index.md:32]: Thanks for the professional explanation, please help to confirm, if yes, I will close this PR.

Comment (dhawton) [reply content/en/docs/tasks/traffic-management/request-timeouts/index.md:32]: It is fine as is. The ) of swapped to `.` in the final render.

Comment (wilsonwu) [reply content/en/docs/tasks/traffic-management/request-timeouts/index.md:32]: Thanks, I will close this change.

**Pull request closed without merging**